torchvision/models/detection/mask_rcnn.py

```diff
@@ -1,6 +1,5 @@
 from collections import OrderedDict
 
-import torch
 from torch import nn
 import torch.nn.functional as F
```

Comment:
I believe that the following two import are also not needed?
`import torch.nn.functional as F` 
`from torchvision.ops import misc as misc_nn_ops`